src/plugins/data_source_management/public/components/popover_button/popover_button.test.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { DataSourceMenuPopoverButton } from './popover_button';
+
+describe('Test on PopoverButton', () => {
```

Comment:
nit: might be worth referencing that it is testing the data source mgmt popover button

Reply:
Got it. Thank you!